app/src/main/res/drawable/bg_pin_view.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/pin_view_selected" android:state_selected="true" />
+    <item android:color="@color/pin_view_default" android:state_selected="false" />
```

Comment:
Since this and the default value are the same, you can simplify this file and remove the `state_selected=false` option